joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -1945,4 +1936,38 @@ class AstCreator(filename: String, global: Global)
   def astForYieldWithOptionalArgumentPrimaryContext(ctx: YieldWithOptionalArgumentPrimaryContext): Seq[Ast] = {
     astForYieldWithOptionalArgumentContext(ctx.yieldWithOptionalArgument())
   }
+
+  def resolveRequireOrLoadPath(argsAst: Seq[Ast]): Seq[Ast] = {
+    val importedNode = argsAst.head.nodes.collectFirst(_.asInstanceOf[NewLiteral]).get
```

Comment:
```suggestion
    argsAst.head.nodes.collectFirst { case x: NewLiteral => x }.foreach { importedNode =>
```

Reply:
Wrap this code a `foreach` so that you don't have to call an unsafe `.get` (which may return an error if there is no `NewLiteral`.

Reply:
yes not using `.get` changed the logic.